uint8 left_arm_status   # 0: 空闲 1: 工作中
uint8 right_arm_status   # 0: 空闲 1: 工作中
run_task/ArmPose left_arm_pose
run_task/ArmPose right_arm_pose